engine/src/main/java/org/hibernate/validator/constraints/br/CNPJ.java

```diff
@@ -45,6 +45,8 @@
 
 	Class<? extends Payload>[] payload() default { };
 
+	boolean alphanumeric() default false;
```

Comment:
Wouldn't an enum here be more future-proof option? something like 
```suggestion
	FormatRevision formatRevision() default FormatRevision.V1;
```
```java
enum FormatRevision {
    // Obviously, the constant names have to be more meaningful 
    //  and connected to the versions of current and new formats.
    V1, V2; 
}
```
This way, if another version is published, it can be added without any new parameters on the constraint annotation...